app/src/main/java/com/sensirion/smartgadget/peripheral/rht_sensor/external/RHTHumigadgetSensorManager.java

```diff
@@ -302,4 +358,44 @@ public void synchronizeDeviceServices(@NonNull final BleDevice device) {
             historyService.synchronizeService();
         }
     }
+
+    /**
+     * Tries to synchronize the services from all the devices.
+     */
+    public void synchronizeServicesFromAllDevices() {
+        Executors.newSingleThreadExecutor().execute(
+                new Runnable() {
+                    @Override
+                    public void run() {
+                        synchronized (RHTHumigadgetSensorManager.this) {
+                            for (final BleDevice device : BleManager.getInstance().getConnectedBleDevices()) {
+                                if (isDeviceRHTNotificationsServicesReady(device)) {
+                                    Log.v(TAG, String.format(
+                                            "synchronizeServicesFromAllDevices -> RHT Service from device %s is synchronized.",
+                                            device.getAddress())
+                                    );
+                                    if (isDeviceSettingsServicesReady(device)) {
+                                        Log.v(TAG, String.format(
+                                                "synchronizeServicesFromAllDevices -> Device settings service from device %s is synchronized",
+                                                device.getAddress())
+                                        );
+                                    } else {
+                                        Log.w(TAG, String.format(
+                                                        "synchronizeServicesFromAllDevices -> RHT Service from device %s is not synchronized yet.",
```

Comment:
"RHT Service" -> should that be "Device settings" ... not ready?
In general I think we have way too much verbosity. So when adding new logs, let's make sure they're actually relevant beyond the debugging of the current patch.